server/concepts/Tournament.ts

```diff
@@ -57,10 +70,12 @@ export default class Tournament implements TournamentWihtoutMS {
     matchId: string,
     currentUserId: string,
   ) {
+    if (data.teamA < 0 || data.teamB < 0) throw new AppError('Score can\'t be negative', 404);
```

Comment:
Hmm score can't be negative and must be integer
Don't use shortcuts, I mean replace `can't` to `can not`

Reply:
Ok